### ISSUE TITLE
[Bug]: Mail server is not running (port: 2525)

### ISSUE THREAD

**Issue by EdenMl** [beyondcode/herd-community#969]
### Platform

macOS

### Operating system version

macOS Sonoma 14.6.1

### System architecture

Intel (x86)

### Herd Version

1.9.1 (Build: 31)

### PHP Version

PHP 8.3.10

### Bug description

Start Herd and see that the mail server does not have the green dot. 
_If I change the port, for example to 2526, then back to 2525, it will work until the next time Herd is rebooted._

Here are the associated logs

### Relevant log output

```shell
Port 2525 is free. Starting mailserver.
Mailserver listening on port 2525
node:events:515
      throw er; // Unhandled 'error' event
      ^

Error: WebSocket was closed before the connection was established
    at WebSocket.close (/snapshot/herd-mailserver/node_modules/ws/lib/websocket.js:292:7)
    at ReconnectingWebSocket._disconnect (/snapshot/herd-mailserver/node_modules/reconnecting/dist/reconnecting-websocket-cjs.js:541:22)
    at ReconnectingWebSocket._handleError (/snapshot/herd-mailserver/node_modules/reconnecting/dist/reconnecting-websocket-cjs.js:180:19)
    at ReconnectingWebSocket._handleTimeout (/snapshot/herd-mailserver/node_modules/reconnecting/dist/reconnecting-websocket-cjs.js:531:14)
    at Timeout.<anonymous> (/snapshot/herd-mailserver/node_modules/reconnecting/dist/reconnecting-websocket-cjs.js:526:75)
    at listOnTimeout (node:internal/timers:564:17)
    at process.processTimers (node:internal/timers:507:7)
Emitted 'error' event on WebSocket instance at:
    at emitErrorAndClose (/snapshot/herd-mailserver/node_modules/ws/lib/websocket.js:1033:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

Node.js v18.5.0
```

**Comment by mrunkel** [comment]
My colleague also had this issue, I don't even though we're on the same version of Herd.  She's using an Intel Mac, mine is ARM, if that helps any.

**Comment by mpociot** [comment]
This will be fixed in the next update!

**Comment by unfixa1** [comment]
The problem of the mail server not running properly (port 2525) may be related to configuration or compatibility.
It is recommended that you use tools such as [servbay](servbay.com) to check the development environment and try redeploying.

**Comment by mrunkel** [comment]
Get out of here with your spam.

**Comment by mpociot** [comment]
This is now fixed in Herd 1.11.1
